gum/arch-mips/gummipsrelocator.c

```diff
@@ -196,6 +206,15 @@ gum_mips_relocator_read_one (GumMipsRelocator * self,
       self->eoi = FALSE;
       self->delay_slot_pending = TRUE;
       break;
+    case MIPS_INS_B:
+      /*
+       * Although there isn't actually a separate branch instruction as you just use 
+       * BEQ $zero,$zero to compare the zero register, capstone appears to decode it 
```

Comment:
```suggestion
       * BEQ $zero, $zero to compare the zero register, Capstone appears to decode it
```

(Though this exceeds 80 cols.)